src/index.html

```diff
@@ -2,13 +2,63 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header class="header">
+      <a href="index.html" class="logo-img">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link is-active" href="index.html">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a class="nav-link" href="index.html">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a class="nav-link" href="index.html">Smartphones</a>
+          </li>
+
+          <li class="nav-item">
+            <a class="nav-link" href="index.html"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a 
              class="nav-link" 
              href="index.html"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
Consider fixing indents according to the code style guide